panther_analysis_tool/main.py

```diff
@@ -305,7 +305,11 @@ def upload_analysis(backend: BackendClient, args: argparse.Namespace) -> Tuple[i
         A tuple of return code and the archive filename.
     """
 
-    if args.batch:
+    supports_async = backend.supports_async_uploads()
+    if supports_async and args.no_async:
+        supports_async = False
```

Comment:
Would something like the following be better:
`use_async = (not args.no_async) and backend.supports_async_uploads()`

and then use `use_async` below?

Reply:
ooh I like that. Will update